src/App.tsx

```diff
@@ -20,12 +25,16 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => {
+                  setQuery(event.target.value);
```

Comment:
You can create separate method and use it onChange, like `onChange={handleSearch}`, where handleSearch is your function:)